evaluator/src/main/kotlin/Rule.kt

```diff
@@ -101,6 +102,35 @@ abstract class Rule(
         ruleMatcherManager.block()
     }
 
+    /**
+     * A [RuleMatcher] that checks whether a [label] exists in the [ORT result][OrtResult.labels]. If [value] is null
+     * the value of the label is ignored.
+     */
+    fun hasLabel(label: String, value: String? = null) =
+        object : RuleMatcher {
+            override val description = "hasLabel(${listOfNotNull(label, value).joinToString()})"
+
+            override fun matches() =
+                if (value == null) {
+                    ruleSet.ortResult.labels.containsKey(label)
```

Comment:
Maybe `label in ruleSet.ortResult.labels`?

Reply:
Done.